CHANGELOG.md

```diff
@@ -3,3 +3,5 @@
 ## Added 
 ### V1.0-SNAPSHOT
 + 2017-10-29 - Initial release.
+### V1.1-SNAPSHOT
++ 2018-09-10 - Bug fixes.
```

Comment:
Can be more precise on the changes, like what is done to enhance the library to handle query params.